boards/arm/lpcxpresso55s69/doc/index.rst

```diff
@@ -202,16 +202,20 @@ see the following message in the terminal:
 Building and flashing secure/non-secure with Arm |reg| TrustZone |reg|
 ----------------------------------------------------------------------
 The TF-M integration samples can be run using the ``lpcxpresso55s69_ns`` target.
-Next we need to manually flash the secure (``tfm_s.hex``)
-and non-secure (``zephyr.hex``) images wth a J-Link as follows:
+To run we need to manually flash the resulting image (``tfm_merged.bin``) with a
+J-Link as follows:
 
-.. code-block:: console
+   .. code-block:: console
+
+      JLinkExe -device lpc55s69 -if swd -speed 2000 -autoconnect 1
+      J-Link>loadfile build/tfm_merged.bin
```

Comment:
Should we add a note on the `r` hack in cases where you can't connect to the debug interface because TF-M has previously started, just to have it on record or explain the common error message?

Reply:
Why are there separate instructions for JLinkExe and west? If it's because of the erase command, you can use `west flash --erase`.

Reply:
@MaureenHelm the west flash command instruction was erased because it's not working as expected.